external/storm-redis/README.md

```diff
@@ -194,6 +195,11 @@ RedisState
                                 fields,
                                 new RedisStateUpdater("test_", tupleMapper).withExpire(86400000),
```

Comment:
@revans2 Yes, right. Seems like I forgot to replace tupleMapper. ;( I'll take care of it.